src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,312 @@
+              </UserSafeContainer>
+
+              {/* Demo Safe Section */}
+              <TryDemoSafe safeAppUrl={safeAppUrl} />
+            </ActionsContainer>
+          </>
+        )}
+      </StyledCard>
+    </Container>
+  )
+}
+
+export default SafeAppLandingPage
+
+const SafeAppDetails = ({ iconUrl, name, description, availableChains }) => {
+  const showAvailableChains = availableChains && availableChains.length > 0
+
+  return (
+    <>
+      <DetailsContainer>
+        <SafeIcon src={iconUrl} />
+        <DescriptionContainer>
+          <SafeAppTitle size="sm">{name}</SafeAppTitle>
+          <div>{description}</div>
+        </DescriptionContainer>
+      </DetailsContainer>
+      <Separator />
+
+      {/* Available chains */}
+      {showAvailableChains && (
+        <>
+          <ChainLabel size="lg">Available networks</ChainLabel>
+          <ChainsContainer>
+            {availableChains.map((chainId) => (
+              <div key={chainId}>
+                <NetworkLabel networkInfo={getChainById(chainId)} />
+              </div>
+            ))}
+          </ChainsContainer>
+          <Separator />
+        </>
+      )}
+    </>
+  )
+}
+
+const CreateNewSafe = () => {
+  return (
+    <>
+      <BodyImage>
+        <Img alt="Vault" height={92} src={SuccessSvg} />
+      </BodyImage>
+
+      <Button size="lg" color="primary" variant="contained" component={Link} to={OPEN_SAFE_ROUTE}>
+        <Text size="xl" color="white">
+          Create new Safe
+        </Text>
+      </Button>
+    </>
+  )
+}
+
+const ConnectWallet = () => {
+  const { clickAway, open, toggle } = useStateHandler()
+
+  return (
+    <StyledProvider>
+      <Provider
+        info={<ProviderDisconnected />}
+        open={open}
+        toggle={toggle}
+        render={(providerRef) =>
+          providerRef.current && (
+            <StyledPopper
+              anchorEl={providerRef.current}
+              open={open}
+              placement="bottom"
+              popperOptions={{ positionFixed: true }}
+            >
+              <ClickAwayListener onClickAway={clickAway} touchEvent={false}>
+                <List component="div">
+                  <ConnectDetails />
+                </List>
+              </ClickAwayListener>
+            </StyledPopper>
+          )
+        }
+      />
+    </StyledProvider>
+  )
+}
+
+const TryDemoSafe = ({ safeAppUrl }) => {
+  return (
+    <SafeDemoContainer>
+      <Title size="xs">Want to try the app before using it?</Title>
+
+      <BodyImage>
+        <Img alt="Demo" height={92} src={DemoSvg} />
+      </BodyImage>
+      {safeAppUrl && (
+        <StyledDemoButton
+          color="primary"
+          component={Link}
+          to={`${demoSafeRoute}?appUrl=${encodeURI(safeAppUrl)}`}
+          size="lg"
+          variant="outlined"
+        >
+          Try Demo
+        </StyledDemoButton>
+      )}
+    </SafeDemoContainer>
+  )
```

Comment:
Thank you for splitting these components. Can you put them in separate folders in the same folder as well?